snapshotter/sys.config

```diff
@@ -105,8 +105,9 @@
    {denylist_url, "https://api.github.com/repos/helium/denylist/releases/latest"},
    {jsonrpc_ip, {127,0,0,1}}, %% bind JSONRPC to localhost only
    {jsonrpc_port, 4467},
+   {rpc_timeout, 300},
```

Comment:
@KevinWassermann94 I think this might help with rpc timeout issue. Did this change only for snapshotter right now. Because I wasn't sure whether it will not have any side effect. Seems good till now.